src/components/inputs/checkbox-input/checkbox-input.js

```diff
@@ -63,41 +98,36 @@ class CheckboxInput extends React.PureComponent {
   });
 
   render() {
+    const disabled = this.props.disabled || this.props.isDisabled;
```

Comment:
Dont we have to do the same for checked?

Reply:
I think it happens in `getCheckboxProps` 🤔

Reply:
maybe could also do

```

const checkboxProps = getCheckboxProps(this.props);
... 
<Label disabled={checkboxProps.disabled}>
```

Reply:
We do, but because it's only used once under `render()`, it is done inline in L117 (apart from also being done in `getCheckboxProps`).

Reply:
yeah, just meant so we could remove the disabled={isDisabled || disabled} thing

Reply:
I like that alternative 👍